src/test/java/com/exasol/javatutorial/markdown/MarkdownStatisticsScalarScriptIT.java

```diff
@@ -0,0 +1,81 @@
+package com.exasol.javatutorial.markdown;
+
+import static com.exasol.matcher.ResultSetStructureMatcher.table;
+import static com.exasol.matcher.TypeMatchMode.NO_JAVA_TYPE_CHECK;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import java.io.FileNotFoundException;
+import java.nio.file.Path;
+import java.sql.*;
+import java.util.concurrent.TimeoutException;
+
+import org.junit.jupiter.api.*;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+import com.exasol.bucketfs.Bucket;
+import com.exasol.bucketfs.BucketAccessException;
+import com.exasol.containers.ExasolContainer;
+import com.exasol.dbbuilder.dialects.DatabaseObjectFactory;
+import com.exasol.dbbuilder.dialects.Schema;
+import com.exasol.dbbuilder.dialects.exasol.ExasolObjectFactory;
+
+@Tag("slow")
+@Testcontainers
+class MarkdownStatisticsScalarScriptIT {
+    private static final String JAR_FILE_NAME = "exasol-java-tutorial.jar";
+    @Container
+    private static final ExasolContainer<? extends ExasolContainer<?>> EXASOL = new ExasolContainer<>().withReuse(true);
+    private static Connection connection;
+    private static DatabaseObjectFactory factory;
+
+    @BeforeAll
+    static void beforeAll() throws SQLException {
+        connection = EXASOL.createConnection();
+        factory = new ExasolObjectFactory(connection);
+    }
+
+    @AfterAll
+    static void afterAll() throws SQLException {
+        if (MarkdownStatisticsScalarScriptIT.connection != null) {
+            MarkdownStatisticsScalarScriptIT.connection.close();
```

Comment:
```suggestion
        if (connection != null) {
            connection.close();
```
Or is this by intention?